stream.go

```diff
@@ -161,6 +173,16 @@ func (cs *clientStream) SendMsg(m interface{}) (err error) {
 
 func (cs *clientStream) RecvMsg(m interface{}) (err error) {
 	err = recv(cs.p, cs.codec, m)
+	defer func() {
+		// err != nil indicates the termination of the stream.
```

Comment:
yes, that's correct.

On Thu, Jun 18, 2015 at 4:54 PM, David Symonds <notifications@github.com>
wrote:

> In stream.go
> <https://github.com/grpc/grpc-go/pull/229#discussion_r32791029>:
>
> > @@ -161,6 +173,16 @@ func (cs *clientStream) SendMsg(m interface{}) (err error) {
> >
> >  func (cs *clientStream) RecvMsg(m interface{}) (err error) {
> >  	err = recv(cs.p, cs.codec, m)
> > +	defer func() {
> > +		// err != nil indicates the termination of the stream.
>
> so it is certain that this method won't be called again after this?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/grpc/grpc-go/pull/229/files#r32791029>.
>